Mathlib/Algebra/GroupWithZero/Action/Defs.lean

```diff
@@ -5,6 +5,7 @@ Authors: Chris Hughes, Yury Kudryashov
 -/
 import Mathlib.Algebra.Group.Action.Defs
 import Mathlib.Algebra.Group.Hom.Defs
+import Mathlib.Tactic.MinImports
```

Comment:
This seems to be the only source of import bumps, I think it's fine either way if we keep this import or remove it.